acceptable/tests/test_openapi.py

```diff
@@ -66,9 +61,87 @@ def test_convert_str_returns_same_value():
         assert "beeblebrox" == result
 
 
+class TidyStringTests(testtools.TestCase):
+    @staticmethod
+    def test_none():
+        assert "None" == openapi.tidy_string(None)
+
+    @staticmethod
+    def test_empty():
+        assert "" == openapi.tidy_string("")
+
+    @staticmethod
+    def test_tidy_returns_tidy():
+        assert "simple" == openapi.tidy_string("simple")
+
+    @staticmethod
+    def test_collapse_two_spaces():
+        assert "hello world" == openapi.tidy_string("hello  world")
+
+    @staticmethod
+    def test_collapse_three_spaces():
+        assert "hello world" == openapi.tidy_string("hello   world")
+
+    @staticmethod
+    def test_strip_trailing_spaces():
+        assert "hello world" == openapi.tidy_string(" hello world ")
+
+
+class GetHeadTests(testtools.TestCase):
+    def test_empty(self):
+        self.assertIsNone(openapi.get_head_of_single_item_list([], "item"))
+
+    @staticmethod
+    def test_single_item_list():
+        assert "HEAD" == openapi.get_head_of_single_item_list(["HEAD"], "item")
+
+    def test_multiple_item_list_returns_none(self):
+        self.assertIsNone(
+            openapi.get_head_of_single_item_list(["HEAD", "TAIL"], "item")
+        )
+
+
+class EndpointToOperationTests(testtools.TestCase):
+    @staticmethod
+    def test_blank():
+        endpoint = AcceptableAPI(
+            introduced_at=1,
+            name="test name",  # maps to operation.operation_id
+            service=AcceptableService(name="test service"),
+            url="https://test.example",
+        )
+        operation = openapi.convert_endpoint_to_operation(endpoint)
+        assert "None" == operation.description
```

Comment:
Why use plain `assert` instead of `self.assertXXX()`? And why are these tests staticmethods instead of plain methods?

Reply:
Let's chat about this.